src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/AbstractEvaluationCommandLineHandler.cs

```diff
@@ -176,6 +195,9 @@ private void ApplyChangesToContext(IProjectChangeDiff difference, IImmutableDict
                 foreach (string includePath in difference.ChangedItems)
                 {
                     UpdateInContextIfPresent(includePath, previousMetadata, currentMetadata, isActiveContext, logger);
+
+                    // TODO: Check for changes in the metadata indicating if we should ignore the file
+                    // in the current configuration.
```

Comment:
FINALLY! I understand. I've asked before if there is some kind of metric used for doing the line breaks on these comments. It seems y'all have a setting that does line breaks at 80 characters. Where is that setting and is there a reason to use 80? Many lines of code are much wider than 80 characters. This method definition is 313 characters wide. Seems like 80 is some kind of archaic value prior to the advent of widescreen monitors.

Reply:
I just manually wrap them at 80. Or sometimes 100. Depends on how I'm feeling that day. :-) I find comment widths in that range to be easier to read.

But we're not really consistent about comment widths.

Reply:
Oh, interesting. I tend to use English language logic for comments. Meaning, if I have several sentences of comments, I'll line break to separate concepts, similar to how paragraphs are used in writing. Otherwise, it looks disorienting to have code lines hundreds of characters wide with skinny comment blocks, IMO. I like line breaks to signify significance, similar to what they do in code (in many cases).

Reply:
I do the samething as yanni. I line break whenever the concept changes or when it starts to look too wide. I don't use any specific width of line breaks anymore than I do for code.